### DISCUSSION TITLE
File/Windows Explorer window opens itself up constantly

### DISCUSSION THREAD

**Discussion by Hoolequin** [valinet/ExplorerPatcher#2607] [Q&A]
How do I stop the File/Windows Explorer window from opening itself up constantly.   It seems to do it randomly, but when it does it is every couple seconds until it feels like stopping.

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.

Please include any other software you have installed that modifies the UI of windows.

This is a known issue in windows: https://www.howtogeek.com/884624/youre-not-alone-windows-11-is-randomly-opening-file-explorer/
